Sources/Action.swift

```diff
@@ -207,6 +207,7 @@ private struct ActionState {
 }
 
 /// A protocol used to constraint `Action` initializers.
+@available(swift, deprecated: 3.1, message: "This protocol is no longer necessary and will be removed in a future version of ReactiveSwift. Use Action directly instead.")
```

Comment:
This would happen when building through Carthage. https://travis-ci.org/ReactiveCocoa/ReactiveSwift/jobs/201869272#L821